apps/client/src/locales/messages.pl.xlf

```diff
@@ -299,7 +299,7 @@
       </trans-unit>
       <trans-unit id="6948811958230386934" datatype="html">
         <source>privacy-policy</source>
-        <target state="new">privacy-policy</target>
+        <target state="translated">polityka-prywatności</target>
```

Comment:
`polityka-prywatnosci` (please remove the special character as it is used as part of the url)

Reply:
Ok, done